src/components/pipelines-overview/utils.ts

```diff
@@ -67,3 +67,18 @@ export const LAST_LANGUAGE_LOCAL_STORAGE_KEY = 'bridge/last-language';
 
 export const getLastLanguage = (): string =>
   localStorage.getItem(LAST_LANGUAGE_LOCAL_STORAGE_KEY);
+
+export const useInterval = (
```

Comment:
Where is the `No refresh` case handled?

Reply:
ya so, in the useInterval hook, when user selects no refresh it updates once, then in this custom hook i have put a condition to run setInterval() only if interval!==null, which is the value when you select no refresh.